test/lib/jdk/test/lib/security/TestCertificate.java

```diff
@@ -41,8 +41,8 @@ public enum TestCertificate {
     // Issuer: CN=Intermediate CA Cert, O=SomeCompany
     // Validity: Tue Aug 30 14:37:19 PDT 2016 to Wed Aug 30 14:37:19 PDT 2017
     ONE("1000",
-        "CN=SSLCertificate, O=SomeCompany",
-        "CN=Intermediate CA Cert, O=SomeCompany",
+        "CN=SSLCertificate,O=SomeCompany",
+        "CN=Intermediate CA Cert,O=SomeCompany",
```

Comment:
Code was modified to use the X509Certificate API (rather than X509CertImpl) 

it's using the X500Principal#getName() call now. Would it be better to use X500Principal#toString() ?

Reply:
fair point. I updated patch to use X500Principal#toString()